### DISCUSSION TITLE
cannot show ONNX graph

### DISCUSSION THREAD

**Discussion by btwbtw01** [Xilinx/finn#554] [Q&A]
Hi,
I want to show ONNX graph by calling showInNetron(), but it can't show. It can't connect. Is the port problem? Or did I miss something?

**Comment by fpjentzsch** [comment]
Hi,
in case you access the notebook not via localhost (e.g. because it runs inside a VM), you need to set the `LOCALHOST_URL` environment variable accordingly.

Alternatively, you can save the .onnx file, go to https://netron.app/, and open it there.

**Reply by btwbtw01** [reply]
Hi, @fpjentzsch 
Thanks for your response. Do you mean that I have to change LOCALHOST_URL IP? I try 172.17.0.1(docker IP address), but it doesn't work.
Here is more information.
![image](https://user-images.githubusercontent.com/85877169/155055067-1aaebe65-3a09-40b1-b2f3-109f9d765b83.png)

**Reply by fpjentzsch** [reply]
If you access the notebook via 172.17.0.1 in your browser, you can set `LOCALHOST_URL=172.17.0.1` before you run `./run-docker.sh notebook`. Then `showInNetron()` should connect to the correct Iframe. You can also provide IP and port as argument to this function (see https://github.com/Xilinx/finn/blob/dev/src/finn/util/visualization.py).
